packages/client/wallets/aa/src/utils/auth.ts

```diff
@@ -11,7 +11,7 @@ export const parseToken = (token: any) => {
             typeof window !== "undefined" ? window.atob(base64) : Buffer.from(base64, "base64").toString();
         return JSON.parse(jsonPayload || "");
     } catch (err) {
-        logError("[PARSE_TOKEN] - ERROR", {
+        logger.logError("[PARSE_TOKEN] - ERROR", {
```

Comment:
but in other instances, this would be fine to do `LoggerProvider.getLogger().logError(...)`